src/components/exchange/ConfirmExchangeButton.js

```diff
@@ -21,7 +22,7 @@ export default function ConfirmExchangeButton({
   disabled,
   loading,
   isHighPriceImpact,
-  insufficientLiquidity,
+  errorCode,
```

Comment:
i'm wondering if `errorCode` is the right name to call it. `errorCode` is an object `QuoteError` that has `error_code` param inside, maybe this can be confusing for new people reading the code (i was a bit confused), WDYT of renaming to quoteError or something like that?

code is working fine btw, i tried it on android and ios 👌